cmd/mpi-operator.v1alpha2/app/server.go

```diff
@@ -17,6 +17,8 @@ package app
 import (
 	"context"
 	"fmt"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
```

Comment:
should you please move these two packages to second part?

Reply:
Sure ;)